### PR TITLE
Hitchhiker galaxy design

### PR DESCRIPTION
**Pull request by codingunicorn**
This is the first change for the Galaxy theme including the background-image. Not connected to a raised issue.

### CODE REVIEW

File: data/initial-data.json [hunk 1]
```diff
@@ -2,7 +2,13 @@
   "partners": [
     {
       "title": "Diamond Partners",
-      "logos": []
+      "logos": [
+        {
+          "logoUrl": "../images/logos/ERNI.png",
```
Comment (lukaselmer) [inline data/initial-data.json]: I suggest to use lower case for filenames 😉

File: docs/tutorials/configure-app.md [hunk 2]
```diff
@@ -59,12 +59,12 @@ window.ENV = 'dev';
...
     "headerSettings": {
       "backgroundColor": "#FF9800",
-      "backgroundImage": "/images/backgrounds/home.png",
+      "backgroundImage": "/images/backgrounds/galaxy-bg.png",
       "fontColor": "#fff",
       "tabBarColor": "#fff",
       // data only applicable for the home page
       "video": {
-        "title": "GDG DevFest Ukraine 2016",
+        "title": "GDG DevFest Ukraine 201",
```
Comment (lukaselmer) [inline docs/tutorials/configure-app.md]: Mm we don't need to change this file, right? Can we revert these changes?

File: src/styles/shared-styles.html [hunk 3]
```diff
@@ -11,9 +11,12 @@
         /* Generate theme at https://www.materialpalette.com */
 
         /* Primary colors */
-        --dark-primary-color:       #007593;
-        --default-primary-color:    #21a4c4;
-        --light-primary-color:      #66d6f7;
+        --dark-primary-color:       #032d54;
+       /*  --default-primary-color:    #21a4c4; */
+        --default-primary-color:   #032d54; 
+        --cta-color:                #80CBC4;
```
Comment (lukaselmer) [inline src/styles/shared-styles.html]: What is this variable used for?

File: src/styles/shared-styles.html [hunk 4]
```diff
@@ -11,9 +11,12 @@
...
+        /* --light-primary-color:      #66d6f7; */
```
Comment (lukaselmer) [inline src/styles/shared-styles.html]: Let's delete the comments - no dead code 😉

File: docs/tutorials/configure-app.md [hunk 5]
```diff
@@ -59,12 +59,12 @@ window.ENV = 'dev';
   "home": {
     "headerSettings": {
       "backgroundColor": "#FF9800",
-      "backgroundImage": "/images/backgrounds/home.png",
+      "backgroundImage": "/images/backgrounds/galaxy-bg.png",
```
Comment (lukaselmer) [inline docs/tutorials/configure-app.md]: The image looks very nice 😊 two things:

* Did you check that it looks good with different screen resolutions?
* The file is nearly 1MB in size, which is quite large. Can you minify it without visible quality loss?

File: data/initial-data.json (hunk 1 above)
Comment (lukaselmer) [reply data/initial-data.json]: Regarding file size: this is super small, nice 😊

File: docs/tutorials/configure-app.md (hunk 5 above)
Comment (codingunicorn) [reply docs/tutorials/configure-app.md]: a) Yes, but I'd want to discuss with you largening the header image anyways, as this is now is causing responsive issues on some devices. 
b) I already minified it to the max.

**Comment by codingunicorn** [comment]
Hey @lukaselmer thanks for your review. I must have overlooked some things in the rush, which I fixed now. I also changed the CTA Color again to the Turquoise from the Style guidelines (see Screenshot below). 


galaxy-theme

Additionally, I added comments to your Background image review. Let me know if this is okay or if you'd want to further minify the image.

**Comment by ChristofBuechi** [comment]
@lukaselmer Is there still some action todo here ? I think the design is merged ? @codingunicorn is waiting for a feedback for this merge request. Whats the status ?
